docs/.vitepress/contributors.ts

```diff
@@ -103,6 +103,22 @@ const plainTeamMembers: CoreTeam[] = [
     title: 'A fullstack developer',
     desc: 'Creating tools for collaboration',
   },
+  {
+    avatar: contributorsAvatars.poyoho,
+    name: 'Yoho Po',
+    github: 'poyoho',
+    twitter: '@yoho_po',
+    title: 'It\'s no problem in my locall',
+    desc: 'Core team member of Vite & Vitest',
+  },
+  {
+    avatar: contributorsAvatars.AriPerkkio,
+    name: 'Ari Perkkiö',
+    github: 'AriPerkkio',
+    twitter: '@perkkio_ari',
```

Comment:
I don't actually have twitter or any other similar platforms at all. Just Github and email. 🙃 

The typings and logic seem to require some refactoring to make this field optional.
```suggestion
```